src/i18n/utils.ts

```
@@ -0,0 +1,169 @@
+    return `/${targetLang}${cleanPath}`;
+  };
+}
+
+/**
+ * Génère les liens hreflang pour le SEO multilingue
+ * @param currentPath - Chemin de la page courante (sans le préfixe de langue)
+ * @param baseUrl - URL de base du site
+ * @returns Array d'objets avec hreflang et href
+ */
+export function getHreflangLinks(currentPath: string, baseUrl: string = '') {
+  const cleanPath = currentPath.startsWith('/') ? currentPath : `/${currentPath}`;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)